src/content/de/updates/2018/01/crux.md

```diff
@@ -10,22 +10,22 @@ description: Announcing the release of a new country dimension in the Chrome Use
 
-Um loszulegen, gehen Sie zum [CrUX-Projekt](https://bigquery.cloud.google.com/dataset/chrome-ux-report:all) auf BigQuery und Sie sehen eine Liste von Datensätzen geordnet nach [Ländercode](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) von `country_ae` (Vereinigte Arabische Emirate) nach `country_za` (Südafrika). Der bekannte `all` -Datensatz ist immer noch vorhanden, um die globalen Gesamtleistungsdaten zu erfassen. Innerhalb jedes Datensatzes gibt es monatliche Tabellen beginnend mit dem letzten Bericht, `201712` . Eine detaillierte Anleitung zum Einstieg finden Sie in unserer aktualisierten [CRUX-Dokumentation](/web/tools/chrome-user-experience-report/).
+Um zu beginnen, gehen Sie zu [CrUX project](https://bigquery.cloud.google.com/dataset/chrome-ux-report:all) auf BigQuery und Sie werden eine Liste von Datensätzen sehen, die von [country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) von `country_ae` (Vereinigte Arabische Emirate) nach `country_za` (Südafrika) organisiert werden. Das vertraute `all` Dataset ist immer noch vorhanden, um die globalen aggregierten Leistungsdaten zu erfassen. Innerhalb jedes Datensatzes gibt es monatliche Tabellen beginnend mit dem letzten Bericht, `201712` . Eine detaillierte Anleitung zum Einstieg finden Sie in unserem aktualisierten [CrUX documentation](/web/tools/chrome-user-experience-report/) .
 
-Wir freuen uns, diese neuen Daten mit Ihnen teilen zu können und hoffen, dass Sie sie nutzen, um die Nutzererfahrung im Web zu verbessern. Um Hilfe zu bekommen, Fragen zu stellen, Feedback zu geben oder Ergebnisse aus Ihrer eigenen Analyse zu teilen, nehmen Sie an der Diskussion im [CRUX forum](https://groups.google.com/a/chromium.org/forum/#!forum/chrome-ux-report) teil. Und wenn die kostenlose Stufe auf BigQuery nicht ausreicht, um Ihre anfragende Begeisterung einzudämmen, führen wir immer noch eine Aktion durch, um Ihnen [zusätzliche 10 TB frei](https://docs.google.com/forms/d/e/1FAIpQLSeMYnz93JQuO7rPewVrKpLfxO7JREOysti0CQyRo31bc7cXHA/viewform) zu geben, also erhalten Sie Ihre Kredite, solange der Vorrat reicht!
+Wir freuen uns, diese neuen Daten mit Ihnen teilen zu können und hoffen, dass Sie sie nutzen, um die Nutzererfahrung im Web zu verbessern. Um Hilfe zu erhalten, Fragen zu stellen, Feedback zu geben oder Ergebnisse aus Ihrer eigenen Analyse zu teilen, nehmen Sie an der Diskussion auf [CrUX forum](https://groups.google.com/a/chromium.org/forum/#!forum/chrome-ux-report) . Und wenn die kostenlose Stufe auf BigQuery nicht ausreicht, um Ihre anfragende Begeisterung einzudämmen, führen wir immer noch eine Promotion durch, um Ihnen eine [extra 10 TB free](https://docs.google.com/forms/d/e/1FAIpQLSeMYnz93JQuO7rPewVrKpLfxO7JREOysti0CQyRo31bc7cXHA/viewform) zu geben. [extra 10 TB free](https://docs.google.com/forms/d/e/1FAIpQLSeMYnz93JQuO7rPewVrKpLfxO7JREOysti0CQyRo31bc7cXHA/viewform) Sie sich also Ihre Credits, solange der Vorrat reicht!
```

Comment:
Space after the link before the period: `nehmen Sie an der Diskussion auf [CrUX forum](https://groups.google.com/a/chromium.org/forum/#!forum/chrome-ux-report) . Und wenn die`